api/lib/domain/models/AuthenticationMethod.js

```diff
@@ -47,7 +46,7 @@ const validationSchema = Joi.object({
   authenticationComplement: Joi.when('identityProvider', [
     { is: identityProviders.PIX, then: Joi.object().instance(PasswordAuthenticationMethod).required() },
     { is: identityProviders.POLE_EMPLOI, then: Joi.object().instance(PoleEmploiAuthenticationComplement).required() },
-    { is: identityProviders.GAR, then: Joi.any().forbidden() },
+    { is: identityProviders.GAR, then: Joi.any().empty() },
```

Comment:
**Question**: Pourquoi ce changement ? La doc sur forbidden dit: "Marks a key as forbidden which will not allow any value except undefined. Used to explicitly forbid keys.". Il me semble que l'on veut justement que la valeur d'`authenticationComplement` pour le provider GAR soit uniquement `undefined`

Reply:
A mon sens l'attribut `authenticationComplement` fait partie du model.
Il peut être `null` ou `undefined`, mais il doit exister.